Marlin/ultralcd_impl_DOGM.h

```diff
@@ -318,10 +318,10 @@ FORCE_INLINE void _draw_heater_status(const uint8_t x, const int8_t heater) {
   #endif
 
   if (PAGE_UNDER(7))
-    _draw_centered_temp((isBed ? thermalManager.degTargetBed() : thermalManager.degTargetHotend(heater)) + 0.5, x, 7);
+    _draw_centered_temp((isBed ? thermalManager.degTargetBed() : thermalManager.degTargetHotend(heater)) + 0.5, x, 7-1);
 
   if (PAGE_CONTAINS(21, 28))
-    _draw_centered_temp((isBed ? thermalManager.degBed() : thermalManager.degHotend(heater)) + 0.5, x, 28);
+    _draw_centered_temp((isBed ? thermalManager.degBed() : thermalManager.degHotend(heater)) + 0.5, x, 28-1);
```

Comment:
The y offset of the glyph is 6, is because of the value yoff of the FONTBOUNDINGBOX in BDF file is set to -2. The height of the glyph is 8, so we have to set the position to (8-2) so that the glyph can be showed in full at the top.